src/components/Text/Text.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import { stylesheet, style } from 'typestyle';
+import { ThemeConsumer, Theme } from '../Theme/Theme';
+/**
+ * Text component to style your `Text` effortlessly.
+ *
+ * **Usage**
+ * ```jsx
+ * <Text>Hello</Text>
+ * ```
+ */
+export const Text: React.FunctionComponent<TextProps> = (props) => {
+  const display = (theme: Theme) => {
```

Comment:
We can move this function in a helper. Also not sure why using theme..
One  thing that would be great if colors can accept theme colors too.
eg
```ts
color: Keyof Theme | string;

color: theme[props.color] || color;
```

Reply:
Only variant thing should happen inside `Text` component. All the styling should stay inside helpers directory.